xsc/services/utils/utils.go

```diff
@@ -0,0 +1,34 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/jfrog/jfrog-client-go/utils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+const (
+	XraySuffix                        = "/xray/"
+	xscSuffix                         = "/xsc/"
+	apiV1Suffix                       = "api/v1"
+	XscInXraySuffix                   = apiV1Suffix + xscSuffix
+	MinXrayVersionXscTransitionToXray = "3.108.0"
+)
+
+// From Xray version 3.108.0, XSC is transitioning to Xray as inner service. This function will return the backward compatibility URL.
```

Comment:
I think the last part of the comment should be :
This function will return compatible URL

Reply:
done